### PR TITLE
add run-mypy.sh

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

**Comment by alexfikl** [comment]
Where is this used?

**Comment by matthiasdiener** [comment]
> Where is this used?

Just added it to CI in https://github.com/inducer/arraycontext/pull/26/commits/2657797694baf7ea792c074368830bd955f97062

**Comment by alexfikl** [comment]
> Just added it to CI in [2657797](https://github.com/inducer/arraycontext/commit/2657797694baf7ea792c074368830bd955f97062)

Ah, fair enough! Add it to the gitlab CI as well?

**Comment by matthiasdiener** [comment]
> > Just added it to CI in [2657797](https://github.com/inducer/arraycontext/commit/2657797694baf7ea792c074368830bd955f97062)
> 
> Ah, fair enough! Add it to the gitlab CI as well?

Good point, thanks! Added in 704272d.

File: run-mypy.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+mypy arraycontext/ examples/ test/
```
Comment (inducer) [inline run-mypy.sh:3]:
```suggestion
python -m mypy arraycontext/ examples/ test/
```

**Pull request merged**